azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -107,3 +107,8 @@ CREATE TABLE IF NOT EXISTS version_set (
 -- TODO: Add the alter table script in the specific release
 -- Adding version_set_id column in execution_flows
 -- alter table execution_flows add column version_set_id INT default null;
+
+-- TODO: Add the alter table script in the specific release
+-- Adding dispatch_method column in execution_flows
+-- alter table execution_flows add column dispatch_method TINYINT default 1;
```

Comment:
Can you please add index for this field? It is in the where clause in select query so it will be useful to have it.

Reply:
done